src/Behat/Behat/Output/Node/EventListener/JUnit/JUnitDurationListener.php

```diff
@@ -83,17 +91,16 @@ private function captureAfterFeatureEvent(Event $event)
         $timer = $this->featureTimerStore[$key];
         if ($timer instanceof Timer) {
             $timer->stop();
-            $this->featureResultStore[$key] = round($timer->getTime());
+            $this->featureResultStore[$key] = $timer->getTime();
```

Comment:
I'm wondering, why do we keep a reference to the timer (after finishing) AND store the timer result in a separate property/array?

- If we want to free up memory, we should `unset($this->featureTimerStore[$key]);` after `$this->featureResultStore[$key] = $timer->getTime();`
- if we don't care about memory, we should remove `featureResultStore` entirely and just call `$timer->getTime()` when needed

CC @higrow

Reply:
@uuf6429 I agree, I can't see a reason to keep both. I doubt the timers take a lot of memory, but we might as well unset them (your first suggestion).

Reply:
I think the 2nd approach would have been more idiomatic.

Otherwise, the first approach would be greatly simplified with direct `microtime(true)` calls.

But just my .2 cents.

(I implemented the first suggestion, so if everyone's fine with that, we can proceed forward)